1-js/10-error-handling/1-try-catch/article.md

```diff
@@ -1,418 +1,417 @@
-# Error handling, "try..catch"
+# Obsługa błędów, "try...catch"
 
-No matter how great we are at programming, sometimes our scripts have errors. They may occur because of our mistakes, an unexpected user input, an erroneous server response, and for a thousand other reasons.
+Nieważne jak świetnymi jesteśmy programistami, zdarza się, że do naszych programów wkradają się błędy. Mogą wystąpić jako rezultat literówki, przez niespodziewane dane wejściowe użytkownika, błędnej odpowiedzi serwera czy z tysiąca wielu innych powodów. 
 
-Usually, a script "dies" (immediately stops) in case of an error, printing it to console.
+Z reguły, nasz program zostaje natychmiastowo przerwany w sytuacji napotkania błędu, zaraz po zwróceniu jego zawartości do konsoli.
 
-But there's a syntax construct `try..catch` that allows to "catch" errors and, instead of dying, do something more reasonable.
+Przy użyciu instrukcji `try...catch`, możemy obsłużyć błędy w bardziej sensowny sposób.
```

Comment:
Fajnie by było uwzględnić jeszcze jakoś ten fragment "instead of dying", to ważne, że przy try...catch nie powodujemy zatrzymania skryptu